README.md

```diff
@@ -200,8 +201,89 @@ would result in
   "name": "the.subobject",
   "field": "field.in.subobject"
 }
+
+```
+### UUID
+An `uuid` transform provides the ability to generate a UUID based on the concat spec. If unable to generate a UUID based on a concat spec, it will failover to the uuid spec, which supports UUID version 3, 4, 5.
+**Note**: version 3 & 5 require an namespace argument. This recursively UUID's multiple fields starting with the namespace passed in.
+```javascript
+{
+    "operation": "uuid",
+    "spec": {
+        "sources": [
+          {"path": "doc.author-id"},
+          {"path": "doc.document-id"},
+          {"path": "doc.meta.id"}
+
+        }],
+        "targetPath": "doc.uuid",
+        "delim": "-"
+    }
+    "uuid": {
+      "version": 5,
+      "nameSpace": "44571214-657c-4f28-9e60-509ec39b22a9" // optional required for version 3 & 5
+
```

Comment:
Should probably also add support for `require` so that behavior can be more tightly controlled when fields are missing.